### DISCUSSION TITLE
Add option to edit relative time ranges on Grafana dashboard

### DISCUSSION THREAD

**Discussion by IamShubhamGupto** [grafana/grafana#46392] [Dashboards, Panels & Visualizations]
**Problem Statement**
We display data that is refreshed only every 24h, once everyday. Having the grafana dashboard show relative time ranges of last 5 minutes, 30 minutes, 6 hours, 12 hours and so on can be misleading to users. They may assume graphs are broken and does not work.

**Proposal**
Allow to editing the list of relative time ranges based on dashboard needs. 

I am aware the same can currently be attached using a variable but the original time range icon is still there and does not get disabled.

**Steps to reproduce**:
Open any grafana dashboard and select the time range button
<img width="405" alt="image" src="https://user-images.githubusercontent.com/32878682/157429525-5582d69a-a43a-48ea-a2d4-29c6a1c2881d.png">

**Actual Result**:
Relative time ranges in minutes, hours, days etc
**Expected Result**
Relative time range only in days

**Environment**:
- Grafana version: v8.3.0 (914fcedb72)
- Data source type & version: ElasticSearch V7+
- User OS & Browser: Windows 10 & Microsoft Edge
- Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
